bin/sbang

```diff
@@ -109,13 +109,17 @@ while read line && ((lines < 2)) ; do
     fi
     lines=$((lines+1))
 done < "$script"
+#this is ineeded for scripts with sbang parameter
+#like ones in intltool
```

Comment:
Sorry, I had a force push so  your suggestion seem outdated.
I added comment but not changed variable names, to be sure to not break anything
If need I can change name accordingly to your spec